test/support/utils.ex

```diff
@@ -140,4 +140,26 @@ defmodule NextLS.Support.Utils do
       assert result == unquote(pattern)
     end
   end
+
+  defmacro did_open(client, file_path, text) do
+    quote do
+      assert :ok ==
+              notify(unquote(client), %{
+                method: "textDocument/didOpen",
+                jsonrpc: "2.0",
+                params: %{
+                  textDocument: %{
+                    uri: uri(unquote(file_path)),
+                    text: unquote(text),
+                    languageId: "elixir",
+                    version: 1
+                  }
+                }
+              })
+    end
+  end
+
+  defmacro did_open(client, file_path) do
+    did_open(client, file_path, File.read!(file_path))
```

Comment:
Yes you're correct but I think I can fix it in the other PR and remove it for now, I will probably just inline the whole other macro at call `File.read!(unquote(file_path))`. I hadn't tested it here since the existing tests that use it just use text instead of real files.